exercises/07-normalizr/src/modules/users/users-saga.ts

```diff
@@ -0,0 +1,47 @@
+import { all, call, fork, put, takeEvery } from 'redux-saga/effects'
+import { AxiosResponse } from 'axios'
+import { normalize } from 'normalizr'
+
+import * as Schema from 'modules/entities/entities-schema'
+import * as EntitiesTypes from 'modules/entities/entities-types'
+
+import { usersActions } from 'modules/users/users-slice'
+import * as UsersEffects from 'modules/users/users-effects'
+import { User } from './user-types'
+
+function* getUsers() {
+  try {
+    const { data: users }: AxiosResponse<User[]> = yield call(
+      UsersEffects.getUsers
+    )
+
+    const normalizedUsers = normalize<
```

Comment:
I think it can't. At least the [documentation](https://github.com/paularmstrong/normalizr/blob/master/docs/api.md#normalizedata-schema) doesn't mention this possibility (on the other hand, the error handling is not described at all).

Reply:
would you mind moving that outside of try block then? apparently only call() can throw

Reply:
There is an explicit `throw` actually. 
 https://github.com/paularmstrong/normalizr/blob/6d32393c602bdbc5f5a8bcdeba6d4b961565abe0/src/index.js#L46

Am I missing something?

Reply:
I can't understand, why is it expected that users look into the implementation to know whether the library function throws? This kind of stuff should be in the documentation. I hope the information is there somwhere :disappointed:

Reply:
I've moved things that can't throw outside of `try/catch`, see  c88a30a